src/try_filter_map.rs

```diff
@@ -174,7 +173,8 @@ where
                     Some(r) => Some(r),
                     None => continue,
                 },
-                v => v,
+                Some(Ok(x)) => Some(Ok(x)),
+                None => None,
```

Comment:
Why was this changed? It doesn't make a difference, does it?

Reply:
It s subtle but `v` is a `Result<T, E>` but the function returns a `Result<T, E2>`.

Reply:
Ah. :laughing: